meilisearch-lib/src/index_controller/mod.rs

```diff
@@ -543,17 +544,14 @@ where
         Ok(stats)
     }
 
-    pub async fn get_all_stats(&self, index_filter: &Option<Vec<String>>) -> Result<Stats> {
+    pub async fn get_all_stats(&self, search_rules: &SearchRules) -> Result<Stats> {
```

Comment:
I'll find a way to not depend on meilisearch auth here later, so that's ok for now